addons/web/__manifest__.py

```diff
@@ -287,6 +287,9 @@
             'web/static/src/legacy/xml/kanban.xml',
             'web/static/src/legacy/xml/search_panel.xml',
             'web/static/src/legacy/xml/week_days.xml',
+
+            # Don't include dark mode files in light mode
+            ('remove', 'web/static/src/search/search_panel/search_view.dark.scss'),
```

Comment:
We don't define dark mode stuff in community :/

Reply:
Weird, somehow without that, it wasn't working. Now it works wo/ it.